### ISSUE TITLE
 Unknown device 

### ISSUE THREAD

**Issue by addonravi** [tananaev/traccar#1588]
Hi tananaev,

I have issue with device imei i putted log below.

can you please help me what was the issue.

2015-12-10 20:23:07  INFO: Query not provided: database.selectDevicesAll
2015-12-10 20:23:07  WARN: Unknown device - 865733024842254

Thanks,
Ravi

**Comment by ivanfmartinez** [comment]
Check you traccar.xml, if you have upgraded you have to add the extra configurations.
your traccar.xml is missing at least something like this :
 
    <entry key='database.selectDevicesAll'>
        SELECT * FROM device;
    </entry>

**Comment by tananaev** [comment]
@addonravi, if you are updating from old version of Traccar, you should read this:

https://www.traccar.org/update-from-2-x-to-3-0/

**Comment by v920749** [comment]
@tananaev  if I add a device in backend directly, does traccar accepts it without restarting the service?

**Comment by tananaev** [comment]
Yes, Traccar will automatically pick up new devices from the database.

**Comment by addonravi** [comment]
Thanks,

its work for me

-Ravi
